### PR TITLE
Add section to security landing page about inhouse networks.

### PR DESCRIPTION
**Pull request by kentonv**
@neynah I wrote text but don't have any ideas about graphics / formatting. Thoughts?

### CODE REVIEW

File: go/security.html [hunk 1]
```diff
@@ -23,6 +23,29 @@
...
+    it takes for hackers to get in and spread across the network.</p>
+
+  <p><strong>Sandstorm protects you.</strong> Fine-grained isolation
+    neutralizes security bugs in apps and prevents attacks from spreading. IT can audit everything
+    that is running and can see which apps are talking to which others.</p>
+
+  <p><strong>Sandstorm also saves time.</strong> On a typical corp network today, an engineer or
+    sys admin may spend hours deploying each app, or weeks waiting for IT. On Sandstorm, it takes
+    one click, and there's no need for IT to review each deployment, because Sandstorm gives IT the
+    tools to stay in control. Plus, developing for Sansdtorm is actually less work for deveolpers,
```
Comment (jparyani) [inline go/security.html]: s/Sansdtorm/Sandstorm/

Comment (jparyani) [inline go/security.html]: s/deveolpers/developers/

**Comment by neynah** [comment]
One possible option:
image

Also, do we want to link "Target got hacked" to an article of some sort?

**Comment by kentonv** [comment]
Done! I'll go ahead and push this so you can see it live, since this isn't being publicized yet anyway.

**Pull request merged**